app/styles/_list-service-instance.less

```diff
@@ -0,0 +1,49 @@
+/*
+  Ideally want a generic "list" that can be used for any unrelated component
+  with enough flexible slots to ensure the layout doesn't break...
+*/
+.flex-row {
```

Comment:
Did you mean to check these flex styles in? I don't see them used except in commented out code.

I'd rather discuss something like this in a separate PR if it's not needed for this one.